roles/openshift_health_checker/openshift_checks/etcd_volume.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for warning about etcd volume size past a defined threshold.
+"""
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class EtcdVolume(OpenShiftCheck):
+    """Ensure disk size for an etcd host does not exceed a defined limit"""
+
+    name = "etcd_volume"
+    tags = ["etcd"]
+
+    etcd_default_size_limit = 0
+
+    def run(self, tmp, task_vars):
+        ansible_mounts = get_var(task_vars, "ansible_mounts")
+
+        etcd_mount_path = self.get_etcd_mount_path(ansible_mounts)
+        etcd_disk_size_available = int(etcd_mount_path["size_available"])
+        etcd_disk_size_total = int(etcd_mount_path["size_total"])
+        etcd_disk_size_used = etcd_disk_size_total - etcd_disk_size_available
+
+        self.etcd_default_size_limit = 0.9 * etcd_disk_size_total
+
+        etcd_disk_size_limit = int(get_var(task_vars, "etcd_max_disk_bytes", default=self.etcd_default_size_limit))
+
+        if etcd_disk_size_used > etcd_disk_size_limit:
+            msg = "Current etcd volume usage ({actual:.2f} GB) is greater than the recommended limit ({limit:.2f} GB)."
```

Comment:
We don't really have a "recommended" limit. probably just "the limit" would be good here. contemplating whether this is also a good place to tell them how to configure the limit.

Also this should mention what the etcd volume turned out to be.